src/main/java/sirius/kernel/xml/XMLCall.java

```diff
@@ -29,6 +29,8 @@
  */
 public class XMLCall {
 
+    private static final String HEADER_CONTENT_TYPE = "Content-Type";
```

Comment:
This constant exists in some apache, google and jsoup classes if any of them is referenced in the kernel.